content/docs/pipelines/sdk/enviroment_variables.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Using enviroment varibales in piplines"
+description = "How to set and use enviroment variables in kubeflow piplines"
++++
+
+This page describes how to create a enviroment varaibale for Kubeflow Pipelines 
+and how they work in a pipline. 
+
+## Before you start
+
+### Initial setup
+Before starting with this tutorial the following things have to be in place: 
+- [Kubeflow set up on kubernetes](https://www.kubeflow.org/docs/gke/deploy/deploy-cli/)
+- Jupyter nooteboks set up on you kubeflow deployment
+
+
+## Using enviroment variables 
+
+Below is a example python lightweight component where we read a enviroment
+variable and the logging it out as a string. 
+
+We are going to use jupyter ligthweight componetes you can find more about them 
```

Comment:
Suggested revision(avoiding we):

Learn more about [building lightweight pipeline components](/docs/pipelines/sdk/lightweight-python-components/).